app/views/layouts/base.html.erb

```diff
@@ -6,7 +6,6 @@
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
 
-    <meta name="robots" content="noindex, nofollow">
```

Comment:
This should only happen for `HostingEnvironment.production?` to avoid staging/QA/sandbox from being indexed.

We should also exclude the support section here, if that's the plan.